model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityExplainInfo.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.apimodel.recordsimilarity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Builder
+@Jacksonized
+@Value
+public class RecordSimilarityExplainInfo {
+    double weight;
+    @JsonProperty(value = "calculated_weight")
```

Comment:
Curious, names with "_" are from a spec?

Reply:
im not totally sure bc i didnt add this, but i believed they are named like this to mirror the rni-es response

Reply:
Yeah @Katsuya-Tomioka , these are from our spec doc (I'll message it to you). If you think we should be using camelCase instead, happy to raise that with the team

Reply:
Actually, I don't think we explicitly wrote these in our doc. we should probably just stick to camelcase everywhere @meganlevin . After that I think this is good to go